apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Adapters.kt

```diff
@@ -93,9 +116,8 @@ val StringAdapter = object  : Adapter<String> {
   }
 }
 
-@SharedImmutable
```

Comment:
Not related to Optionals, but removed these, I think they're no longer useful with the new MM?

Reply:
Good catch!